File: URI_1072.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{
    int n,x,count=0,flag=0;
    cin>>n;
    for( int i=1; i<=n; i++ )
    {
        cin>>x;
        if( x>=10 && x<=20 )
        {
            count++ ;
        }
        else
        {
            flag++ ;
        }
    }
    cout<<count<<" in"<<endl;
    cout<<flag<<" out"<<endl;
}

File: URI_1072.java
import java.io.IOException;
 
import java.util.Scanner;
public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner scanner =new Scanner(System.in);
		int n = scanner.nextInt();
		int x,count=0,flag=0;
	    for( int i=1; i<=n; i++ )
	    {
	    	x = scanner.nextInt();
	        if( x>=10 && x<=20 )
	        {
	            count++ ;
	        }
	        else
	        {
	            flag++ ;
	        }
	    }
	    System.out.printf("%d in\n",count);
	  System.out.printf("%d out\n",flag);
 
    }
 
}
